tests/css/csscomb_test_files/.csscomb.json

```diff
@@ -0,0 +1,4 @@
+{
+  "space-after-opening-brace": 1,
+  "space-before-closing-brace": 1
+}
```

Comment:
Somewhere at the back of my mind I had the newline fear, but failed to review myself. My bad.